File: DevExpress-Demo/VB/RichEditDemo.Wpf/Modules/FirstLook/PageInfoConverter.vb
Imports System
Imports System.Windows.Data

Namespace RichEditDemo

    Public Class PageInfoConverter
        Implements IMultiValueConverter

        Public Function Convert(ByVal values As Object(), ByVal targetType As Type, ByVal parameter As Object, ByVal culture As Globalization.CultureInfo) As Object Implements IMultiValueConverter.Convert
            Return String.Format("PAGE: {0} OF {1}", values(0), values(1))
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetTypes As Type(), ByVal parameter As Object, ByVal culture As Globalization.CultureInfo) As Object() Implements IMultiValueConverter.ConvertBack
            Throw New NotImplementedException()
        End Function
    End Class
End Namespace

File: DevExpress-Demo/CS/RichEditDemo.Wpf/Modules/FirstLook/PageInfoConverter.cs
using System;
using System.Windows.Data;

namespace RichEditDemo {
    public class PageInfoConverter : IMultiValueConverter {
        public object Convert(object[] values, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            return String.Format("PAGE: {0} OF {1}", values[0], values[1]);
        }
        public object[] ConvertBack(object value, Type[] targetTypes, object parameter, System.Globalization.CultureInfo culture) {
            throw new NotImplementedException();
        }
    }
}
